assets/src/edit-story/components/popup/index.js

```diff
@@ -93,11 +101,22 @@ function Popup({
       }
       setPopupState({
         offset: getOffset(placement, spacing, anchor, dock, popup),
+        height: popup.current?.getBoundingClientRect()?.height,
       });
     },
     [anchor, dock, placement, spacing, mounted]
   );
 
+  useEffect(() => {
+    // If the popup height changes meanwhile, let's update the popup, too.
+    if (
+      popupState?.height &&
+      popupState.height !== popup.current?.getBoundingClientRect()?.height
+    ) {
+      positionPopup();
+    }
+  }, [popupState?.height, positionPopup]);
```

Comment:
This (obviously) doesn't trigger if the popup resizes itself. E.g. if the popup is shown at the very bottom of the screen and you switch to "All stories" in the dropdown with multiple lines of colors. Then the popup will be cut off until you scroll in the design panel.

It's of course hard to fix that generally, but maybe we could add a manual function, that the popup could invoke from within asking the parent to reposition it?

It's just a nit - this PR already solves 99% of the existing problems so it's quite the edge case:

![colorpicker-resizing-bug](https://user-images.githubusercontent.com/637548/112226722-e8e4d200-8c04-11eb-96e7-e74fa1ac989b.gif)

Reply:
Added a fix in 4bc2c8e. It feels a bit hacky since it's for this case specifically but it seems to work. Let me know what you think 🐾